### ISSUE TITLE
ClassNotFoundException after importing library

### ISSUE THREAD

**Issue by MaaxGr** [patrickfav/bcrypt#38]
After i have added this library to gradle:
```
implementation group: 'at.favre.lib', name: 'bcrypt', version: '0.9.0'
```

build the Project into a JAR and run it
```
java -jar build/libs/example-0.0.1.jar
```

i get the following error:
```
Error: Could not find or load main class io.ktor.server.netty.EngineMain
Caused by: java.lang.ClassNotFoundException: io.ktor.server.netty.EngineMain
```

If i comment out only your dependency everything is fine.
Seems to be connected to issue #30. If i manually delete everything out of the META-INF Folder (except MANIFEST.MF) everything works again.

**Comment by sidneywidmer** [comment]
I have the same problem :point_up:

**Comment by MaaxGr** [comment]
I have a workaround for this: Download the jars from mavencentral and delete the META-INF Folder. Than throw the library into a libs folder and add the jar from cradle via the local file. Hope it helps @sidneywidmer

**Comment by sidneywidmer** [comment]
@MaaxGr Thx for the hint, that worked :+1: But would be nice if there was a cleaner way.

**Comment by MaaxGr** [comment]
Yes of course. But just in case you need it know (as I did 😅)

**Comment by LSafer** [comment]
I think your problem is not from this library rather the usage of Gradle!

Adding `implementation` is not enough to make Gradle add the library to the output jar.
You might use https://github.com/johnrengelman/shadow or search with the keyword `fat jar`

**Comment by MaaxGr** [comment]
Yeah. I assumed that everyone in this issue knows how to build a fat-jar :D
Nevertheless it don't works for me. Have you tested your solution @LSafer?

**Comment by LSafer** [comment]
No, I didn't 😅

I thought you didn't knew about fat-jars but It seems that the issue isn't about it

About the issue, it might be because there is a dependency of 'bcrypt' itself missing at runtime.
